t/core/request.t

```diff
@@ -437,3 +437,31 @@ c=z_z&v=x%20x
 nil
 --- error_log
 the post form is too large: request body in temp file not supported
+
+
+
+=== TEST 13: get_method
+--- config
+    location = /hello {
+        client_body_in_file_only clean;
```

Comment:
We don't need this directive, right?

Reply:
https://github.com/apache/apisix/blob/93b33d8f5f4af73c621d52d017d114127593b936/apisix/plugins/openwhisk.lua#L72

This function is used here, and when I needed to use it, I realized that it was not wrapped in our core library, which would mix up the use of `ngx.req` and `core.request`, so I added this function and its tests.

Reply:
`client_body_in_file_only` is not relative to get_method.

Reply:
removed